### PR TITLE
v6.1 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -4,13 +4,143 @@
...
+ - Known issues related to the Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - Adding an at-mention at the start of a post draft and pressing the leftwards or rightwards arrow can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+
+### Contributors
```
Comment (amyblais) [inline source/install/self-managed-changelog.md:141]: @puerco The below list of contributors also includes a list of translators that Carrie generated from Weblate. Carrie was also able to identify the URLs to most of the translators's Weblate profiles', except a few of them which you can see in this preview https://github.com/mattermost/docs/blob/126e2d409e8cc78aee86e0e3105770a2a851eaf5/source/install/self-managed-changelog.md#contributors.

Do you have admin-level access in Weblate to find the missing profile links? Please DM me for any questions.

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -4,13 +4,143 @@
...
+    - Fixed an issue with missing threads in the Thread list.
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Added ``EnableInlineLatex`` to add support for inline Latex rendering.
+ - Under ``JobSettings`` in ``config.json``
+   - Added ``CleanupJobsThresholdDays``. This defines the time gap in hours beyond which older jobs will be removed. Default is -1 which means the feature is disabled. Setting to 0 will clean all completed jobs.
```
Comment (agnivade) [inline source/install/self-managed-changelog.md:107]: time gap in "days"

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -4,13 +4,143 @@
...
+
+### Improvements
+
+#### User Interface (UI)
+ - Added one-click reactions for posts. Also, the three most recently used emojis will display when the mouse is hovered on a post.
+ - Added support for selecting names and aliases in the emoji picker.
+ - Changed the user interface of the edit-indicator of posts and moved it inline.
+ - The updated "Tips & Next Steps" screen is now shown to all System Admins.
+ - Once the user has selected **Start Trial**, they will see a modal that lists all of the features now available to them through the Enterprise plan.
+ - Once a non-licensed server has reached 10 users, a modal is displayed to the admin encouraging them to start a 30-day trial.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:41]:
```suggestion
 - Once a non-licensed server has reached 10 users, a modal is displayed to System Admins encouraging them to start a 30-day trial.
```

File: source/install/self-managed-changelog.md [hunk 4]
```diff
@@ -4,13 +4,143 @@
...
+
+### Bug Fixes
+ - Fixed a broken link to the **Custom Emoji** page on servers with a subpath configured.
+ - Fixed an issue where a "No results found" error string was displayed in the **Direct Messages** modal.
+ - Fixed an issue where the caret was placed in the middle of the emojis when picking two emojis from the emoji picker.
+ - Fixed an issue where **System Console > Channels > Channel Management** displayed an option to toggle group management in Team Edition, Starter, and Professional.
+ - Fixed an issue where the channel switcher was missing the "(You)" indicator on the user's own Direct Message channel.
+ - Fixed an issue where the clock format set by the user was not respected on the edit indicator popover.
+ - Replaced Metropolis font files with a new set to correct a kerning issue.
+ - Fixed an issue where deep links opened on mobile displayed an incorrect text directing users to open the Desktop app.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:80]:
```suggestion
 - Fixed an issue where deep links opened on mobile displayed an incorrect message directing users to open the Desktop app.
```

File: source/install/self-managed-changelog.md [hunk 5]
```diff
@@ -4,13 +4,143 @@
...
+ - Addressed various user interface style bugs from v6.0 release.
+ - Fixed emails templates for clients that do not support the ``style`` tag.
+ - Fixed an issue where the scrollbar was hardly visible with Denim & Sapphire Themes.
+ - Fixed an issue where creating a bot with an invalid username returned an "invalid email" error.
+ - Fixed an issue where using ``/code`` did not render initial whitespace characters.
+ - Fixed an issue where **Try Enterprise for Free** option was missing spacing in mobile webview.
+ - Fixed an issue where the SQLStore cache was relied on when populating the WebConn channel memberships.
+ - Fixed an issue where logging was not re-configured when the server config was changed via the System Console.
+ - Fixed an issue with an indigo theme glitch when returning from Playbooks to Channels.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:89]:
```suggestion
 - Fixed a display issue with the Indigo theme when returning from Playbooks to Channels.
```

File: source/install/self-managed-changelog.md [hunk 6]
```diff
@@ -4,13 +4,143 @@
...
+ - Fixed an issue where a "No results found" error string was displayed in the **Direct Messages** modal.
+ - Fixed an issue where the caret was placed in the middle of the emojis when picking two emojis from the emoji picker.
+ - Fixed an issue where **System Console > Channels > Channel Management** displayed an option to toggle group management in Team Edition, Starter, and Professional.
+ - Fixed an issue where the channel switcher was missing the "(You)" indicator on the user's own Direct Message channel.
+ - Fixed an issue where the clock format set by the user was not respected on the edit indicator popover.
+ - Replaced Metropolis font files with a new set to correct a kerning issue.
+ - Fixed an issue where deep links opened on mobile displayed an incorrect text directing users to open the Desktop app.
+ - Addressed various user interface style bugs from v6.0 release.
+ - Fixed emails templates for clients that do not support the ``style`` tag.
+ - Fixed an issue where the scrollbar was hardly visible with Denim & Sapphire Themes.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:83]:
```suggestion
 - Fixed an issue where the scrollbar was hardly visible with Denim & Sapphire themes.
```

File: source/install/self-managed-changelog.md [hunk 7]
```diff
@@ -4,13 +4,143 @@
...
+ - Fixed an issue where using ``/code`` did not render initial whitespace characters.
+ - Fixed an issue where **Try Enterprise for Free** option was missing spacing in mobile webview.
+ - Fixed an issue where the SQLStore cache was relied on when populating the WebConn channel memberships.
+ - Fixed an issue where logging was not re-configured when the server config was changed via the System Console.
+ - Fixed an issue with an indigo theme glitch when returning from Playbooks to Channels.
+ - Fixed an issue where the offline indicator color did not use the correct theme color.
+ - Fixed various bugs for the Collapsed Reply Threads (Beta) feature, including:
+    - Fixed an issue where the recent sidebar sorting option didn't only consider parent posts.
+    - Fixed an issue where a badge was displayed on a thread list when the thread was started by another user in a Direct Message.
+    - Fixed an issue where the user avatar was displayed in the participants list after their post was deleted and if they had no other posts in the thread.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:94]:
```suggestion
    - Fixed an issue where the user avatar was displayed in the participants list after their post was deleted when the user had no other posts in the thread.
```

File: source/install/self-managed-changelog.md [hunk 8]
```diff
@@ -4,13 +4,143 @@
...
+    - Fixed an issue where the ephemeral message was not displyaed as the centre post.
+    - Fixed an issue with dragging and dropping files on a thread while on a Threads panel.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:96]:
```suggestion
    - Fixed an issue with dragging and dropping files on a thread while on the Threads panel.
```

File: source/install/self-managed-changelog.md [hunk 9]
```diff
@@ -4,13 +4,143 @@
...
+    - Fixed an issue where permalinks were not highlighting a post on a thread that was already open on the right-hand side.
+    - Fixed an issue with missing threads in the Thread list.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:98]:
```suggestion
    - Fixed an issue with missing threads in the Threads list.
```

File: source/install/self-managed-changelog.md [hunk 10]
```diff
@@ -4,13 +4,143 @@
...
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.0, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Timed Do Not Disturb
+ - Added the ability to disable all notifications for a specified period of time to avoid distractions, without losing important messages when you're back.
+
+#### Cross-team Recent Mentions
+ - Recent mentions and saved posts now show across all teams.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:31]: Searches as well, correct, given the changes to POST /api/v4/posts/search documented later on this page?

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

File: source/install/self-managed-changelog.md (hunk 10 above)
Comment (amyblais) [reply source/install/self-managed-changelog.md:31]: @larkox Can you help confirm?

File: source/install/self-managed-changelog.md [hunk 11]
```diff
@@ -4,13 +4,143 @@
...
+
+### Improvements
+
+#### User Interface (UI)
+ - Added one-click reactions for posts. A user's three most recently used emojis display when the user hovers over a message.
+ - Added support for selecting names and aliases in the emoji picker.
+ - Changed the user interface of the edit-indicator of posts and moved it inline.
+ - The updated "Tips & Next Steps" screen is now shown to all System Admins.
+ - Once the user has selected **Start Trial**, they will see a modal that lists all of the features now available to them through the Enterprise plan.
```
Comment (jasonblais) [inline source/install/self-managed-changelog.md:40]: Propose moving these under "Administration" or "Enterprise Edition"
```
 - Once the user has selected **Start Trial**, they will see a modal that lists all of the features now available to them through the Enterprise plan.
 - Once a non-licensed server has reached 10 users, a modal is displayed to System Admins encouraging them to start a 30-day trial.
```

File: source/install/self-managed-changelog.md [hunk 12]
```diff
@@ -4,13 +4,143 @@
...
+
+#### Administration
+ - Bulk imports with attached files now log and continue when a file fails to upload instead of halting.
+ - ``get flagged posts`` endpoint will now return only flagged posts for channels the user is a member of.
+ - Updated Bleve to v2 to use the scorch index type.
+ - Minimum supported browser versions changes:
+   - Chrome updated from ``61+`` to ``89+``.
+   - Firefox updated from ``60+`` to ``78+``.
+   - MacOS updated from ``10.9+`` to ``10.14+``.
+ - Prometheus metrics are now enabled when running a standalone jobserver.
```
Comment (jasonblais) [inline source/install/self-managed-changelog.md:70]: I think this applies to Enterprise Edition only? (Enterprise plan)

**Review by jasonblais — CHANGES_REQUESTED** [review]
Two small notes, else LGTM!

**Review by agnivade — APPROVED** [review]

File: source/install/self-managed-changelog.md (hunk 1 above)
Comment (puerco) [reply source/install/self-managed-changelog.md:141]: I think I can complete the list by getting the GitHub logins from weblate. Do you want me to push a commit to your branch?

File: source/install/self-managed-changelog.md (hunk 10 above)
Comment (larkox) [reply source/install/self-managed-changelog.md:31]: Searches are limited to the current team by the user agent (webapp/mobile). There are plans to include that in the future, but not yet.

Comment (larkox) [reply source/install/self-managed-changelog.md:31]: cc @esethna

File: source/install/self-managed-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/install/self-managed-changelog.md:141]: @puerco I think that should be fine, thank you

Comment (puerco) [reply source/install/self-managed-changelog.md:143]: Ready @amyblais I've managed to complete all links except one:  Erik Pfeiffer

**Review by puerco — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Review by spirosoik — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

**Review by chenilim — APPROVED** [review]
Boards update notes LGTM!

File: source/install/self-managed-changelog.md [hunk 13]
```diff
@@ -4,13 +4,156 @@
...
+ - ``get flagged posts`` endpoint will now return only flagged posts for channels the user is a member of.
+ - Updated Bleve to v2 to use the scorch index type.
+ - Minimum supported browser versions changes:
+   - Chrome updated from ``61+`` to ``89+``.
+   - Firefox updated from ``60+`` to ``78+``.
+   - MacOS updated from ``10.9+`` to ``10.14+``.
+
+#### Enterprise Edition
+ - Once the user has selected **Start Trial**, they will see a modal that lists all of the features now available to them through the Enterprise plan.
+ - Once a non-licensed server has reached 10 users, a modal is displayed to System Admins encouraging them to start a 30-day trial.
```
Comment (JtheBAB) [inline source/install/self-managed-changelog.md:81]: Already here: https://github.com/mattermost/docs/pull/5234/files#diff-eba248b2c1a66750343693c8d84e6909fc3c26f299880afcc7261837a0465323R49

Comment (jasonblais) [inline source/install/self-managed-changelog.md:81]:
```suggestion
 - Once a non-licensed server has reached 10 users, a one-time modal is displayed to System Admins encouraging them to start a 30-day trial.
```

**Pull request merged**